packages/base/src/client.ts

```diff
@@ -303,7 +344,11 @@ export class AppKit {
     AssetUtil.getConnectorImage(connector)
 
   // -- Private ------------------------------------------------------------------
-  private async initControllers(options: AppKitOptions) {
+  private async initControllers(
+    options: AppKitOptions & {
```

Comment:
Why remove adapters from type and add it manually here?

Reply:
Bc we need to pass different types to the `adapters` array since the v5 is only have single adapter. And that's how we are fixing the missing exposed functions that I explained in the PR description. So our backwards compatible versions will be sharing same types between adapter <-> base.